rdflib/plugins/stores/concurrent.py

```diff
@@ -20,6 +20,7 @@ def __next__(self):
         return next(self.gen)
 
 
+# NOTE: This does not seem to be tested and probably should inherit from Store.
```

Comment:
It could - and probably should - be removed completely - it is neither context- nor formula-aware so can only be of extremely limited use.